components/elm/cime_config/testdefs/testmods_dirs/elm/extrasnowlayers/user_nl_elm

```diff
@@ -0,0 +1 @@
+use_extrasnowlayers = .true.
```

Comment:
Is this setting a default value in the default namelist? If so, it should probably be .false. 
We want the old snowpack model to be used by default for now or there will likely be non BFB changes wherever there is snow on the sfc. We can easily toggle this on as we test / improve on the SMB.

Reply:
This is in a testmod so we can turn it on by using a configuration like:
```
SMS.ne30_oECv3_gis.IGELM_SIA.elm-extrasnowlayers
```
but otherwise have it off by default. We could also turn it on using user_nl_elm by hand

Reply:
thanks for the clarification